tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -277,6 +277,13 @@ - (void)application:(UIApplication*)application didReceiveProtocol:(WFUri*)uri {
                 return S_OK;
             });
```

Comment:
No, ToString is also used.

---
In reply to: [112285907](https://github.com/Microsoft/WinObjC/pull/2569#discussion_r112285907) [](ancestors = 112285907)